src/main.rs

```diff
@@ -2,6 +2,7 @@
 //! brings `farmer` and `node` together
 
 #![deny(missing_docs)]
+#![feature(concat_idents)]
```

Comment:
Is it used?

Reply:
yes, without this, compiler gets angry about unstable feature. Because I stole `generate_builder!` macro from sdk 😎